src/web/app/components/comment-box/comment-edit-form/comment-edit-form.component.spec.ts

```diff
@@ -42,4 +44,116 @@ describe('CommentEditFormComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  // toggleVisibilityTable
+  it('should initialize with the visibility table collapsed', () => {
+    expect(component.isVisibilityTableExpanded).toBeFalsy();
+  });
+
+  it('should toggle visibility table from collapsed to expanded', () => {
+    expect(component.isVisibilityTableExpanded).toBeFalsy();
+    component.toggleVisibilityTable();
+    // After toggling, it should be expanded
+    expect(component.isVisibilityTableExpanded).toBeTruthy();
+  });
+
+  it('should toggle visibility table from expanded to collapsed', () => {
+    component.isVisibilityTableExpanded = true;
+    expect(component.isVisibilityTableExpanded).toBeTruthy();
+    component.toggleVisibilityTable();
+    // After toggling, it should be collapsed
+    expect(component.isVisibilityTableExpanded).toBeFalsy();
+  });
+
+  // triggerModelChange
+  it('should trigger model change with updated field', () => {
+    const testField = 'commentText';
+    const testData = 'Updated comment text';
+
+    const emitSpy = jest.spyOn(component.modelChange, 'emit');
+
+    component.triggerModelChange(testField, testData);
+
+    // Verify that the model should be updated
```

Comment:
nit: these verify and check comments could be removed as well, as the assertions tell us what they are checking already